tornadox/blocks/step_impl/ek0_diagonal.py

```diff
@@ -0,0 +1,111 @@
+"""EK0 with independent dimensions."""
+
+from functools import partial
+
+import jax
+import jax.numpy as jnp
+
+from tornadox.blocks import sqrtutil
+
+
+@partial(jax.jit, static_argnames=("f",))
+def attempt_step_forward_only(*, f, m, c_sqrtm, p, p_inv, a, q_sqrtm):
+    """A step with the 'BatchEK0'.
```

Comment:
The files suggest it's called
```suggestion
    """A step with the 'DiagonalEK0'.
```
?

Reply:
Oops. You're right.